pkg/eventprocessor/impl/eventprocessor.go

```diff
@@ -87,6 +89,15 @@ func New(
 		return nil, fmt.Errorf("initializing metric instruments: %s", err)
 	}
 
+	if config.WebhookURL != "" {
+		whe, err := NewWebhook(config.WebhookURL)
+		if err != nil {
+			return nil, fmt.Errorf("webhook endpoint cannot be initialized: %s", err)
```

Comment:
so, these are the cases when there is a URL configured, but it is "invalid" (unexpected hostname, malformed URL, etc). 

i think there are two cases. one, where the validator had accidentally put in the wrong URL. In that case, they might want to know immediately, so "silent failing" might not be good. in the other case, they might be trying to deliberately put in a wrong value, but i am not sure why they would do that. they could just remove the URL from the config. so, i think it might be okay not to let them start the validator and force them to fix the URL. What do you think?

Reply:
sounds good! The validator will be initialized, it's just the event processor for that chain with misconfigured URL that will not

Reply:
yes, i meant the validator will start, but then it will `exit` when it tries to configure the `chainStack` [here](https://github.com/tablelandnetwork/go-tableland/pull/572/files#diff-ae22ad1e20b89eaefa8e09e4ed89443ac10fdc555b9767625f3866e3853100afR229), right?

Reply:
oh yeah, you're right!